### DISCUSSION TITLE
What is the key 'travel' distance

### DISCUSSION THREAD

**Discussion by lathoub** [rrainey/virtual-agc-mechanical-cad#9] [Q&A]
How much travel distance did the keys have?
a) from non-pressed to fully pressed.
b) from non-pressed to relay click

I have made my first working key, and have measured
a) 4 mm
b) 3 mm (feel like too much)

**Comment by rrainey** [comment]
There is probably a testing/acceptance document for the push button switch assembly that specifies tolerances. Perhaps one is among the scanned NARA aperture cards (Boxes 473-477 on [this page](https://www.ibiblio.org/apollo/ElectroMechanical.html).  One aggravation here is that we have so far not located a drawing with the specs for the push button's leaf spring, which would really determine much of the play in the pressing action.  We also lack a drawing for the key cap housing assembly, so we don't even have a part number for the leaf spring. The 2003875 drawing indicates the gap between the leaf spring and the actual switch is about 0.12 inches. There would still be some movement until the switch activates and there's further play before the switch bottoms-out.  Again, without precise specifications for the leaf spring (and the travel of the switch), we'd be forced to make educated guesses.

**Comment by lathoub** [comment]
Thanks!
I found [this picture](http://i.imgur.com/Ed7MWt8.jpg) that I can use to measure and extrapolate the distances!

**Reply by lathoub** [reply]
Taken from here: https://deskthority.net/viewtopic.php?t=17251

**Comment by rrainey** [comment]
What estimates did you end up with?

**Reply by lathoub** [reply]
No measurement just yet - I did found out that the picture taken above comes from a [Fran Blanche video](https://www.youtube.com/watch?v=UjcfepTdvZI&t=19s&ab_channel=FranBlanche) and the keys comes up around 46:13 - additional info I can use to measure (probably over the weekend)

On another topic, I fought these [micro switches from Farnell](https://be.farnell.com/multicomp/m140t01-ae0502a/microswitch-spdt-5a-lever/dp/1735351) to work well.

Tomorrow bespoke springs will come in, made to order with the specs from the scanned sheets (3.3LBS/Inch).
